docs/superpowers/specs/2026-06-21-binance-demo-scalping-review-flow-design.md

```
@@ -0,0 +1,110 @@
+```python
+binance_demo_scalping_review_flow_enabled: bool = False
+```
+(env `BINANCE_DEMO_SCALPING_REVIEW_FLOW_ENABLED`, pydantic case-insensitive. 기본 False = default-off.)
+
+### 3.3 (이 PR 아님) Prefect 배포 등록
+
+robin-prefect-automations에 deployment 등록, **paused-by-default**. **권장 cron: 매일 ~23:55 UTC** (그날 거래 완료 + 1d 캔들 거의 완성 시점 반영). 운영자가 (a) `BINANCE_DEMO_SCALPING_REVIEW_FLOW_ENABLED=true` 설정 + (b) deployment unpause 시 자동 동작.
+
+## 4. 데이터 흐름
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)